### DISCUSSION TITLE
Uncaught Error: failed to load /ammo/ammo.wasm.js

### DISCUSSION THREAD

**Discussion by zxopink** [enable3d/enable3d#250] [Help / Questions] [answered]
When I make a new project I always get this error, been sitting on it for few hours and couldn't figure out a solution.
This is my dependencies list:

    "@enable3d/ammo-physics": "^0.25.1",
    "@tweenjs/tween.js": "^18.6.4",
    "@types/matter-js": "~0.17.6",
    "@types/three": "~0.133.1",
    "ammojs-typed": "^1.0.6",
    "enable3d": "^0.24.1",
    "matter-js": "~0.17.1",
    "three": "~0.133.1"

**Comment by yandeu** [comment, marked as answer]
You have to copy the ammo files (kripken or moz) manually.
https://github.com/enable3d/enable3d-website/tree/master/src/lib/ammo

**Reply by zxopink** [reply]
And where should I put the ammo folder? this is my project:
![image](https://user-images.githubusercontent.com/30203588/155337154-3e106d50-18b8-4b46-8ec1-975656f79f88.png)

**Reply by yandeu** [reply]
It doesn't matter where. Just any public folder. The point the `physicsLoader` to it.

**Reply by zxopink** [reply]
Fixed!
I was using the `Live Server` VS Code extension for my testing and forgot to set the `/dist` file as root in the settings so ammo was never loaded to the browser and enable3d couldn't find it

**Reply by yandeu** [reply]
👍🏻

Btw, give [Five Server](https://marketplace.visualstudio.com/items?itemName=yandeu.five-server) a try.
